espr/tests/type_decl.rs

```diff
@@ -0,0 +1,83 @@
+use espr::{ast::SyntaxTree, codegen::rust::*, ir::IR};
+
+const EXPRESS: &str = r#"
+SCHEMA test_schema;
+  TYPE a = STRING;
+  END_TYPE;
+
+  TYPE b = ENUMERATION OF (
+      are,
+      sore,
+      dore
+    );
+  END_TYPE;
+
+  ENTITY c_entity;
+    x: REAL;
+  END_ENTITY;
+
+  TYPE c = c_entity;
+  END_TYPE;
```

Comment:
What did you try to test by this components? It seems that this part is not related to this PR.

Reply:
This is not a direct change, but since we have changed the specification for type_decl, we need to add a test to see if there are any changes to the existing items. In other words, the use of placeholders for entity renaming should be tested here. If anything, the missing point is that I have not tested whether placeholders are used when renaming a or b. I will add that now.